scenarios/kubernetes_kops_aws.py

```diff
@@ -121,6 +129,22 @@ def main(args):
         service = '/service-account.json'
         cmd.extend(['-v', '%s:%s:ro' % (args.service_account, service),
                     '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
+    profile = args.aws_profile
+    if args.aws_role_arn:
+        with tempfile.NamedTemporaryFile(
+            prefix='aws-config', delete=False) as cfg:
+            cfg.write(ROLE_CONFIG.format(
+                role_arn=args.aws_role_arn,
+                source_profile=profile))
+            cmd.extend([
+              '-v', '%s:%s:ro' % (cfg.name, aws_config),
+              '-e', 'AWS_SDK_LOAD_CONFIG=true',
+            ])
+        profile = 'jenkins-assumed-role' # From ROLE_CONFIG
+    cmd.extend([
```

Comment:
Yeah, the standard credentials file uses only a default profile (and it's unnamed).